{ 'fall_forward': { 'head': [],
                    'left_arm': [ { 'duration': 0.027,
                                    'name': u'fall_forward',
                                    'positions': [ -0.1067,
                                                   0.7563,
                                                   -0.6351,
                                                   -0.0213,
                                                   0.0975,
                                                   1.5311,
                                                   -0.019],
                                    'starttime': 0.0}],
                    'left_leg': [ { 'duration': 0.028,
                                    'name': u'fall_forward',
                                    'positions': [ -0.01,
                                                   -0.0943,
                                                   -1.5698,
                                                   2.49,
                                                   0.9111,
                                                   -0.0785],
                                    'starttime': 0.0}],
                    'right_arm': [ { 'duration': 0.027,
                                     'name': u'fall_forward',
                                     'positions': [ 0.1067,
                                                    -0.7563,
                                                    0.6351,
                                                    0.0213,
                                                    -0.0975,
                                                    -1.5311,
                                                    0.019],
                                     'starttime': 0.0}],
                    'right_leg': [ { 'duration': 0.03,
                                     'name': u'fall_forward',
                                     'positions': [ 0.01,
                                                    0.0943,
                                                    1.5698,
                                                    -2.49,
                                                    -0.9111,
                                                    0.0785],
                                     'starttime': 0.0}],
                    'torso': []}}
